modules/apps/client-extension/client-extension-service/src/main/java/com/liferay/client/extension/internal/service/taglib/ClientExtensionBottomJSPDynamicInclude.java

```diff
@@ -1,5 +1,5 @@
 /**
- * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-FileCopyrightText: (c) 2023 Liferay, Inc. https://liferay.com
```

Comment:
This isn't right. Not sure how it happened, but I remember seeing on slack that someone else also had issues with the copyright updating to 2023, maybe through their editor or something

Reply:
Mmm, I'm relying on the SF to fix these things and the editor doesn't change anything, AFAIK. Strange...

Reply:
Here was the past discussion I saw on it https://liferay.slack.com/archives/C1SJ64S9X/p1694676523871839